completions/bash/brew

```diff
@@ -563,10 +563,10 @@ _brew() {
 
   if [[ "$i" -eq "$COMP_CWORD" ]]
   then
-    # Do not auto-complete "instal" abbreviation for "install" command.
+    # Do not auto-complete "*instal" abbreviations for "*install" commands.
     # Prefix newline to prevent not checking the first command.
-    local cmds=$'\n'"$(brew commands --quiet --include-aliases)"
-    __brewcomp "${cmds/$'\n'instal$'\n'/$'\n'}"
+    local cmds=$'\n'"$(brew commands --quiet --include-aliases | grep -v instal$)"
```

Comment:
I could get it to work without calling grep was:
```
local cmds=$'\n'"$(brew commands --quiet --include-aliases)"
cmds="${cmds/$'\n'uninstal$'\n'/$'\n'}"
__brewcomp "${cmds/$'\n'instal$'\n'/$'\n'}"
```
I could not figure out a regex pattern that would work to capture both instal and uninstal in a single line, and went for the one liner (which I also find more readable).